CCS2.0/DataLibrary/DataAccess/SqlDataAccess.cs

```diff
@@ -34,5 +34,13 @@ public static int SaveData<T>(string sql, T data)
                 return cnn.Execute(sql, data);
             }
         }
+
+        internal static List<T> LoadData<T>(string sql)
```

Comment:
You could make `parameters` an optional argument.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments